recipes/configparser/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "3.5.0b2" %}
+
+package:
+    name: configparser
+    version: {{ version }}
+
+source:
+    fn: configparser-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/c/configparser/configparser-{{ version }}.tar.gz
+    md5: ad2a71db8bd9a017ed4735eac7acfa07
+
+build:
+    number: 0
+    skip: True  # [py35]
+    script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
So, unfortunately, we still have to use `bld.bat` because of [this]( https://github.com/conda-forge/staged-recipes/pull/293#issuecomment-207526417 ). Once that changes we can go back and drop them, but we stille need them for now.